support-frontend/assets/pages/paper-subscription-checkout/components/thankYou.jsx

```diff
@@ -104,12 +120,12 @@ function ThankYouContent({
             </Text>
           )
         }
-        {startDate &&
+        {(startDate && !hideStartDate) &&
           <Text title={fulfilmentOption === HomeDelivery ? 'You will receive your newspapers from' : 'You can start using your vouchers from'}>
```

Comment:
Is the 'You can start using your vouchers from' copy ok for Subscription Card?

Reply:
That entire section with the start date is hidden when using the card- `hideStartDate` will be true if `useDigitalVoucher` is true and the fulfillment option is Collection.